Sources/TuistKit/Commands/UpCommand.swift

```diff
@@ -16,64 +16,46 @@ class UpCommand: NSObject, Command {
     /// File handler instance to interact with the file system.
     fileprivate let fileHandler: FileHandling
 
-    /// Printer instance to output information to the user.
-    fileprivate let printer: Printing
-
-    /// Graph loader instance to load the project and its dependencies.
-    fileprivate let graphLoader: GraphLoading
-
-    /// Graph up instance to print a warning if the environment is not configured.
-    fileprivate let graphUp: GraphUpping
-
     /// Path to the project directory.
     let pathArgument: OptionArgument<String>
 
+    private let setupLoader: SetupLoading
```

Comment:
I'd add a documentation line explaining what this attribute is for. Something like:

```
Instance to load the setup manifest and perform the project setup
```

Reply:
Ah, missed that. Updated - many thanks for the suggestion.